### DISCUSSION TITLE
Per client: local network access

### DISCUSSION THREAD

**Discussion by esiqveland** [wg-easy/wg-easy#188] [Ideas]
Maybe you do not want to give every client full access to your local network, but only some. It would be neat if this could be configured per client in the web UI. Is this something that could be doable?

**Comment by psycmos** [comment]
Check IP address network, mine 10.11.12.2
Check you IP address of wireguard network, mine: 10.8.0.0/24
Replace with your peers, mine ( 10.8.0.2, etc)

That rule enable access to port clients(10.8.0.2,10.8.0.4,10.8.0.5) for 2 ports ( 1234 and port 4321)
The admin is 10.8.0.3, that client have all access garanted

Create a variable called WG_POST_UP:
iptables -t nat -I POSTROUTING -o eth0 -j MASQUERADE -s 10.8.0.0/24;
iptables -N WIREGUARD_wg0;
iptables -A FORWARD -j WIREGUARD_wg0;
iptables -A WIREGUARD_wg0 -o wg0 -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT;
iptables -A WIREGUARD_wg0 -s 10.8.0.3 -i wg0 -j ACCEPT;
iptables -A WIREGUARD_wg0 -s 10.8.0.2 -i wg0 -d 10.11.12.2 -p tcp --dport 1234:4321 -j ACCEPT;
iptables -A WIREGUARD_wg0 -s 10.8.0.4 -i wg0 -d 10.11.12.2 -p tcp --dport 1234:4321 -j ACCEPT;
iptables -A WIREGUARD_wg0 -s 10.8.0.5 -i wg0 -d 10.11.12.2 -p tcp --dport 1234:4321 -j ACCEPT;
iptables -A WIREGUARD_wg0 -i wg0 -j DROP;
iptables -A WIREGUARD_wg0 -j RETURN;

Create a variable called WG_POST_DOWN
iptables -t nat -D POSTROUTING -o eth0 -j MASQUERADE -s 10.8.0.0/24;
iptables -D FORWARD -j WIREGUARD_wg0;
iptables -F WIREGUARD_wg0;
iptables -X WIREGUARD_wg0;
